substrate/frame/support/test/tests/construct_runtime_ui/deprecated_where_block.stderr

```diff
@@ -347,7 +347,7 @@ error[E0277]: the trait bound `Runtime: Config` is not satisfied
 26 |         System: frame_system::{Pallet, Call, Storage, Config<T>, Event<T>},
    |         ^^^^^^ the trait `Config` is not implemented for `Runtime`
    |
-note: required by a bound in `frame_system::GenesisConfig`
```

Comment:
Did you mean to remove this one instead since you have removed `GenesisConfig`?

Reply:
This test result was generated by `./scripts/update-ui-tests.sh 1.75`